src/main/java/com/ixortalk/autoconfigure/oauth2/auth0/mgmt/api/Auth0Users.java

```diff
@@ -69,23 +59,22 @@ public void createBlockedUser(String email, String password, String firstName, S
                 roleNamesToAssign.stream().map(roleName -> this.auth0ManagementAPI.listRolesByName().get(roleName).getId()).collect(toList()));
     }
 
-    public String createEmailVerificationTicket(String email, String resultUrl, int timeToLiveInSeconds) {
-        return
-                ofNullable(auth0ManagementAPI.listUsersByEmail().get(email))
+    public String createEmailVerificationTicket(String userId, String resultUrl, int timeToLiveInSeconds) {
+        return auth0ManagementAPI.getUserById(userId)
                         .map(user -> auth0ManagementAPI.createEmailVerificationTicket(user.getId(), resultUrl, timeToLiveInSeconds))
-                        .orElseThrow(() -> new Auth0RuntimeException("User with email '" + email + "' not found"));
+                        .orElseThrow(() -> new Auth0RuntimeException("User with id '" + userId + "' not found"));
     }
 
     public void unblockUser(String email) {
```

Comment:
Maybe better to throw an exception here as well when the user cannot be found?  It was not the case before either, but think it's best to deal with this consistently?